core/src/scss/core/_base.scss

```diff
@@ -33,3 +33,7 @@ body {
 [hidden] {
   display: none !important;
 }
+
+figure {
```

Comment:
This belongs in elements.

Reply:
Hmmmm.....I don't remember adding this. I did merged in the aspect ratio PR so probably from there.